yafti/app.py

```diff
@@ -16,6 +16,10 @@
 
 import hashlib
 
+## ---- TODO: START: REMOVE THIS HACK AFTER ENOUGH PEOPLE HAVE UPGRADED ----
+from pathlib import Path
+## ---- TODO: END: REMOVE THIS HACK AFTER ENOUGH PEOPLE HAVE UPGRADED ----
```

Comment:
These signposts are a bit aggressive. Instead I suggested how I'd like to see this tracked on the snippet below.

```suggestion
from pathlib import Path
```